core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -710,7 +713,9 @@ map_processor_fold(#kz_binding{binding_parts=BParts
                   ,_Routing
                   ,RoutingParts
                   ) ->
-    case matches(BParts, RoutingParts) of
+    case matches(BParts, RoutingParts)
+        orelse matches(RoutingParts, BParts)
```

Comment:
i agree on that proposal and in fact, after a closer look, we misused bindings in tasks. in `tasks`, modules should bind to `tasks.help` and implement proper arity's for `help` and the `tasks.help` usage in `tasks_scheduler` should use `fold` instead of `map` passing [], [Category] or [Category, Action] so proper arity would be called. it should also pass the acumulator in the first param, so help/0 in modules should become help/1, help/2, help/3.

for the bindind queries, we could use matches/1 with the `orelse` implemented only there.